torchaudio/sox_effects/sox_effects.py

```diff
@@ -260,7 +261,7 @@ def apply_effects_file(
 
 
 @_mod_utils.requires_module('torchaudio._torchaudio')
-@_mod_utils.deprecated('Please migrate to `apply_effects_file` or `apply_effects_tensor`.')
+@_mod_utils.deprecated('Please migrate to `apply_effects_file` or `apply_effects_tensor`.', "0.8.0")
 def SoxEffect():
     r"""Create an object for passing sox effect information between python and c++
```

Comment:
nit: no need to change the docstring though we did discuss removing warnings from the docstring to avoid duplicated signal